pkg/webhooks/networkpolicies/networkpolicies.go

```diff
@@ -26,7 +26,7 @@ const (
 
 var (
 	timeout                          int32 = 2
-	allowedUsers                           = []string{"system:admin", "backplane-cluster-admin"}
+	allowedUsers                           = []string{"system:admin", "backplane-cluster-admin", "system:serviceaccounts:openshift-ocm-agent-operator:ocm-agent-operator"}
 	sreAdminGroups                         = []string{"system:serviceaccounts:openshift-backplane-srep"}
 	privilegedServiceAccountGroupsRe       = regexp.MustCompile(privilegedServiceAccountGroups)
```

Comment:
The `system:serviceaccounts:openshift-ocm-agent-operator:ocm-agent-operator` should be part of this already? So might be a bug?

Reply:
It must be a bug :thinking:
Probably best to look into why exactly this isn't working in the first place cc @dakotalongRH